composer.json

```
@@ -26,6 +26,9 @@
         "symfony/phpunit-bridge": "^5.2",
         "vimeo/psalm": "^5.13"
     },
+    "replace": {
+        "mongodb/builder": ">= 0.2"
```

Since the library includes all version of the builder and we never really advertised it, I think we should be able to replace all versions here. WDYT?
```suggestion
        "mongodb/builder": "*"
```

---

Ok, that's fine.